app/views/errors/internal_server_error.html.haml

```diff
@@ -0,0 +1,12 @@
+.d-flex.align-items-center.flex-column.position-relative
+    = inline_svg("errors/agroportal.svg") 
+    .position-absolute{style: "top: 442px; font-size: 30px; font-weight: 800; color: #263238;"} 
+        Something went wrong. We have been notified about this error.
```

Comment:
the old message was better: `We're sorry but something has gone wrong. We have been notified of this error.`